todo-list/src/App.vue

```diff
@@ -16,12 +16,12 @@
         <th align="left">状態</th>
       </thead>
       <tbody>
-        <tr v-for="(task, index) in tasks" v-bind:key="task.id">
+        <tr v-for="(task, index) in tasks" v-bind:key="task.id" :class="[{'none':task.stateNum === 1 && filter === 'complete'},{'none':task.stateNum === 0 && filter === 'work'}]">
```

Comment:
`,`の後ろ、`:`の後ろと`{}`の内側両サイドにはスペースを空けましょう。
v-bindの省略記法と通常の記法が混在しているので、統一したほうが違和感がないかと思います。
```suggestion
        <tr v-for="(task, index) in tasks" :key="task.id" :class="[{ 'none': task.stateNum === 1 && filter === 'complete' }, { 'none': task.stateNum === 0 && filter === 'work' }]">
```